pullrequest.rst

```diff
@@ -119,10 +119,30 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you will need to enter this command::
```

Comment:
I don't sure if we were using git description? If yes, we are using description, I think we should mention `git commit`, because this will open the editor and let user to enter title and description.

If not, I think using `git commit -m` is fine.